components/button/demo/button.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta name="viewport" content="width=device-width, initial-scale=1.0">
+		<meta http-equiv="X-UA-Compatible" content="ie=edge">
+		<meta charset="UTF-8">
+		<link rel="stylesheet" href="../../demo/styles.css" type="text/css">
+		<script src="/node_modules/@webcomponents/webcomponentsjs/webcomponents-loader.js"></script>
+		<script type="module">
+			import '../../demo/demo-page.js';
+			import '../button.js';
+		</script>
+	</head>
+	<body unresolved>
+
+		<d2l-demo-page title="d2l-button">
+			<h3>Button</h3>
+			<d2l-demo-snippet>
+				<d2l-button>Normal Button</d2l-button>
+			</d2l-demo-snippet>
+			<h3>Button (Primary)</h3>
+			<d2l-demo-snippet>
+				<d2l-button primary="">Primary Button</d2l-button>
+			</d2l-demo-snippet>
+			<h3>Button (Disabled)</h3>
+			<d2l-demo-snippet>
+				<d2l-button disabled="">Disabled Button</d2l-button>
+			</d2l-demo-snippet>
+			<h3>Button (Primary and Disabled)</h3>
+			<d2l-demo-snippet>
+				<d2l-button disabled="" primary="">Disabled Primary Button</d2l-button>
+			</d2l-demo-snippet>
+		</d2l-demo-page>
+
+		<script>
+			const id = setInterval(() => {
+				const buttons = document.querySelectorAll('d2l-button');
+				if (buttons.length === 4) {
+					clearInterval(id);
+					for (let i = 0; i < buttons.length; i++) {
+						buttons[i].addEventListener('click', (e) => {
+							// eslint-disable-next-line no-console
+							console.log('Button Clicked', e);
+						});
+					}
+				}
+			}, 50);
```

Comment:
@dbatiste, @margaree, @svanherk: do we still care about this? Doesn't seem like something that's particularly useful in a demo.

Reply:
I don't think we need this imo.  At one point I think we saw clicks handlers being triggered (on the component, not the native button) for disabled buttons, and this may have been helpful to debug that.  If we really want to keep it, we can probably just wire up a handler on the body with 1 or 2 lines of code.

Reply:
@cjmorrison-d2l sounds like we can kill this little script